spec/rspec/abq_spec.rb

```diff
@@ -88,7 +92,10 @@ def stringify_keys(hash)
       EnvHelper.with_env("ABQ_SOCKET" => "#{host}:#{port}") do
         example.call
       end
-      RSpec::Abq.instance_eval { @socket = nil }
```

Comment:
Is this an unrelated change?

Reply:
sorry updated the PR description to reflect the actual changes

overall "fix tests on main"

Reply:
This is expanded below by removing all instance variables. That's probably what should have been here before since we use a handful of of them in the Abq singleton class.